tests/test-org-ql.el

```diff
@@ -2111,7 +2111,84 @@ with keyword arg NOW in PLIST."
+              (spy-on 'org-agenda-files :and-return-value '())
+              (expect (org-ql-view--contract-buffers-files temp-filenames) :to-equal temp-filenames))
+            (it "buffer"
+              (with-current-buffer (org-ql-test-data-buffer "data.org")
+                (expect (org-ql-view--contract-buffers-files (current-buffer)) :to-equal "buffer")))
+            (it "org-agenda-files from symbol"
+              (spy-on 'org-agenda-files :and-return-value temp-filenames)
+              (expect (org-ql-view--contract-buffers-files 'org-agenda-files) :to-equal "org-agenda-files")
+              (expect (org-ql-view--contract-buffers-files #'org-agenda-files) :to-equal "org-agenda-files"))
+            (it "Non specific list"
+              (let ((value1 '("a.org" "b.org"))
+                    (value2 'a))
+                (expect (org-ql-view--contract-buffers-files value1) :to-equal value1)
+                (expect (org-ql-view--contract-buffers-files value2) :to-equal value2))))
+          (describe "Expanding org-ql-view-buffers-files"
+            (it "all"
+              (let* ((buffer-names (list (make-temp-file "test" nil ".org") (make-temp-file "test" nil ".other")))
+                     (buffers (mapcar #'get-buffer-create buffer-names)))
+                (save-excursion
+                  (switch-to-buffer (car buffers))
+                  (org-mode))
+                (spy-on 'buffer-list :and-return-value buffers)
+                (expect (org-ql-view--expand-buffers-files "all") :to-equal (list (car buffers)))))
+            (it "org-agenda-files"
+              (spy-on 'org-agenda-files :and-return-value "value for org-agenda-files")
+              (expect (org-ql-view--expand-buffers-files "org-agenda-files") :to-equal "value for org-agenda-files"))
+            (it "org-directory"
+              (spy-on 'org-ql-search-directories-files :and-return-value "value for org-directory")
+              (expect (org-ql-view--expand-buffers-files "org-directory") :to-equal "value for org-directory"))
+            (it "buffer"
+              (with-temp-buffer
+                (expect (org-ql-view--expand-buffers-files "buffer") :to-equal (current-buffer))))
+            (it "literal values"
+              (with-temp-buffer
+                (expect (org-ql-view--expand-buffers-files (current-buffer)) :to-equal (current-buffer)))
+              (let ((test-buffer (get-buffer-create (make-temp-file "test"))))
+                (expect (org-ql-view--expand-buffers-files test-buffer) :to-equal test-buffer))
+              (let ((test-list '(1 2 3)))
+                (expect (org-ql-view--expand-buffers-files test-list) :to-equal test-list))
+              (let ((string-literal "this is a string"))
+                (expect (org-ql-view--expand-buffers-files string-literal) :to-equal string-literal))))
+          (describe "Testing org-ql-view--complete-buffers-files"
```

Comment:
Sometimes it seems awkward, but it would generally be good to write these test labels in the idiomatic Buttercup way, so that they read as sentence clauses in the output.  I'm not sure I've been entirely consistent with this, myself, but when done well, it makes test output much easier to understand.

Reply:
Adding this to the descriptions